apps/src/code-studio/components/AdvancedShareOptions.jsx

```diff
@@ -404,12 +433,22 @@ class AdvancedShareOptions extends React.Component {
         ShareOptions.EMBED,
         this.props.i18n.t('project.embed')
       );
+      if (
+        experiments.isEnabled(experiments.STUDENT_LIBRARIES) ||
+        librariesEnabled
```

Comment:
Just to be sure, `librariesEnabled` will be false for the general public at this point, right?

Reply:
Yep